readme.md

```diff
@@ -49,19 +49,19 @@ docker run -it --rm -p 8006:8006 --device=/dev/kvm --cap-add NET_ADMIN --stop-ti
 
 ## FAQ
 
-* ### How do I use it?
+- ### How do I use it?
 
   Very simple! These are the steps:
-  
+
   - Start the container and connect to [port 8006](http://localhost:8006) using your web browser.
```

Comment:
Hii

Reply:
add the --env=VERSION=x
which x is the windows version, and the list is available in the readme